src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let sum;
+
+  if (days >= 1 && days < 3) {
+    sum = 40 * days;
+  }
+
+  if (days >= 3 && days < 7) {
```

Comment:
You can use `else if` instead of separate `if` statements to make it clear that these conditions are mutually exclusive. This also avoids unnecessary condition checks if a previous condition has already been met.